tidb-cloud/terraform-use-dedicated-network-container-resource.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Use TiDB Cloud Dedicated Network Container Resource
+summary: Learn how to use the TiDB Cloud Dedicated network container resource to create and modify a TiDB Cloud Dedicated network container.
+---
+
+# Use TiDB Cloud Dedicated Network Container Resource
+
+This document introduces how to manage a [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) network container using the `tidbcloud_dedicated_network_container` resource.
+
```

Comment:
What is network container and why users need it? It is not explained in other TiDB Cloud docs. It will be user friendly to add a short introduction in this doc?